3-UsingAPIs/Week2/homework/ex4-diceRace.js

```diff
@@ -12,14 +13,17 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 
 function rollTheDices() {
   const dices = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDice()
+  return Promise.race(dices.map((dice) => rollDice(dice)));
```

Comment:
Good!

Even shorter would be:

```suggestion
  return Promise.race(dices.map((rollDice));
```